Car3_Aux.py

```diff
@@ -0,0 +1,30 @@
+import ctypes
+import random
+from DAD.DAD import DAD
+from Mocks.Steering import AuxSteering
+import time
+import threading
+
+# Configure DAD board to emmulate the CAN bus
+emulator = DAD()
+emulator.CAN_init(baudRate=500e3)
+# Turn on 5v to power the CAN tranciver
+emulator.posSupply_init()
+mc = Mitsuba(0x08F89540, 0x01, emulator)
+mppt = Proton1(1024, 0x2, emulator)
+bms = Orion(0x6B0, 0x1, emulator)
+steering = AuxSteering(1, 1, emulator)
+
+auxThread = threading.Timer(1, steering.sendCAN)
+auxThread.daemon = True
+auxThread.start()
+
+bmsThread = threading.Timer(0.1, bms.sendCAN)
+bmsThread.daemon = True
+bmsThread.start()
+
+try:
+    while True:
+        emulator.receiveCAN(cb)
```

Comment:
Change this to just receive the Aux message so either above do a cb = {steering.receive msg} (check the mitsuba mock line 102:
```
def receiveCAN(self, data):
        print("Mitsuba Data: "+(" ".join("0x{:02x}".format(c) for c in data)))
```
And [this](https://www.geeksforgeeks.org/print-lists-in-python-4-different-ways/) on how to print lists in python.

Reply:
I'll admit that I tried to look at the mitsuba file but it confused me more than anything. I wasn't sure what I was supposed to print a list for. So, I just assigned cb to a dictionary like it was in Car3.py. I tested that Car3_Aux runs successfully this time, but I can't do more than see if it runs because I don't have a DAD board.